readme.md

```diff
@@ -11,3 +11,10 @@ Welcome to the MIT Libraries Developer Documentation site.
 While this site is public and you are welcome to use it, our main goals are
 internally facing. That said, feel free to open an Issue if you have suggestions
 or feedback.
+
+## Side Nav Ordering
```

Comment:
👍  Thanks - this was not clear to me when I interacted with this system previously and it is nice to have clarification.